### PR TITLE
Query related_image table when rendering DC from sqlite

### PR DESCRIPTION
**Pull request by joelanford**
**Description of the change:**
When rendering declarative config from an sqlite database, query the `related_image` table to find any related images we haven't already found from the CSVs in the `operatorbundle` table.

**Motivation for the change:**
Typically, the `operatorbundle` table contains CSVs only for channel heads. Therefore querying _just_ the CSVs for related images will typically result in finding related images for channel heads.

To find all related images, the `related_image` table should also be queried.

Why not use _just_ the `related_image` table? Because the CSV's `relatedImages` field has an image name and image reference, whereas the `related_image` table contains just the image reference. The declarative config's `olm.bundle` `relatedImages` field allows a `name` to be set, so we use the `related_image` table to supplement what we find the the CSVs.  

**Reviewer Checklist**
- [x] Implementation matches the proposed design, or proposal is updated to match implementation
- [x] Sufficient unit test coverage 
- [ ] Sufficient end-to-end test coverage
- [ ] Docs updated or added to `/docs` 
- [x] Commit messages sensible and descriptive

### CODE REVIEW

File: internal/action/render.go [hunk 1]
```diff
@@ -165,9 +166,55 @@ func sqliteToDeclcfg(ctx context.Context, dbFile string) (*declcfg.DeclarativeCo
...
 	return &cfg, nil
 }
 
+func populateDBRelatedImages(ctx context.Context, cfg *declcfg.DeclarativeConfig, db *sql.DB) error {
+	rows, err := db.QueryContext(ctx, "SELECT image, operatorbundle_name FROM related_image")
+	if err != nil {
+		return err
+	}
+	images := map[string]sets.String{}
+	for rows.Next() {
```
Comment (timflannagan) [inline internal/action/render.go:182]: I was blank on whether you need to defer the rows.Close() here. Looking at the godocs, it looks like it should be safe assuming rows.Next() eventually returns false, as the rows will get closed automatically.

Comment (joelanford) [reply internal/action/render.go:182]: Good catch. Can't hurt to do it explicitly.

Comment (joelanford) [reply internal/action/render.go:185]: Done!

**Pull request merged**